pkg/xattr/xattr.go

```diff
@@ -14,6 +14,7 @@ const (
 var cap_empty_bitmask = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
 var supported_capabilities = map[string][]byte{
 	"cap_net_bind_service": {1, 0, 0, 2, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+	"cap_sys_ptrace":       {1, 0, 0, 2, 0, 0, 8, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
```

Comment:
Do you think you can add a minimal tar file which contains files with these attrs, load it and check if it matches?

Reply:
Do you mean as a functional test? I'm not sure how to do that in here.

Reply:
There is a test in xattr_test.go already which reads a checked in tar archive. That archive can just container handcrafted files via system tools. We can then compare our results to it (like done for the selinux part there).